code/game/objects/items/devices/paicard.dm

```diff
@@ -132,6 +134,8 @@
 				pai.laws.set_zeroth_law("Serve your master.")
 				pai.emittersemicd = FALSE
 		if("set_laws")
+			if(!in_range(src, usr))
```

Comment:
this needs to go AFTER the stripped_multiline_input

Reply:
done